inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -254,6 +261,12 @@ public function delete_used_css_on_update_or_delete( $post_id ) {
 			return;
 		}
 
+		$deletion_activated = $this->is_rucss_deletion_activated();
+
+		if ( ! $deletion_activated ) {
+			return;
+		}
```

Comment:
```suggestion
		$deletion_activated = $this->is_rucss_deletion_activated();

		if ( ! $deletion_activated ) {
			return;
		}
```
```suggestion
		if ( ! $this->is_rucss_deletion_activated() ) {
			return;
		}
```

Same for the other cases, it's not necessary to assign to a variable here